README.md

```diff
@@ -122,6 +122,16 @@ The installation directory for Rundeck.
 ##### `server_web_context`
 
 Web context path to use, such as "/rundeck". `http://host.domain:port/server_web_context`
+( Use this for Rundeck versions < 3.3 )
+
+##### `server_web_context33`
```

Comment:
IMO it will be better to add optional `compatibility` param which accepts a version (can be `SemVer` type) and defaults `undef` . Then do condition when using `server_web_context`. This will avoid confusion with 2 different `params` which have same purpose. Eventually old versions will be deprecated and preserving same param will avoid breaking change.